Automattic/newspack-blocks
# Update Subscribe patterns for WordPress 5.9

### All Submissions:


### Changes proposed in this Pull Request:

In WordPress 5.9, the Cover block has the ability to "smartly" figure out whether the image in it is dark or light and change the text colour accordingly; unfortunately, it does not get this quite right for the Subscribe block patterns, and the text does not have sufficient contrast.

This PR makes the text in Subscribe pattern #2 and #3 white to fix this.

Closes #1014

### How to test the changes in this Pull Request:

1. Start with a test site running the WordPress 5.9 RC.
2. Open the pattern library and insert Subscribe patterns #2 and #3 in the editor; note both have dark text against a pretty dark Cover block image, both in the editor and in the pattern preview:

![image](https://user-images.githubusercontent.com/177561/149683841-5c1394ba-486f-4a6e-9611-1f4b4faa6c94.png)

3. Apply the PR and run `npm run build`.
4. Reopen the pattern preview and reinsert the blocks; confirm that the heading text is now white and legible, and that no errors are thrown:

![image](https://user-images.githubusercontent.com/177561/149683813-bdfe950e-066f-4537-bb9c-9f4d42e74c83.png)


### Other information:
